src/rpn/rpn.ts

```diff
@@ -1,31 +1,57 @@
 export function rpn(inputString: string): any {
+    console.log(inputString)
     if (inputString.length === 420) throw new Error("Blaze it");
+    if ( /[a-zA-Z]/i.test(inputString)) throw new Error("Invalid Expression");
 
-    const operandsAndOperators: Array<number | string> = inputString.split(" ").map((token) => {
-        var parsedToken = isNaN(Number(token))
+    const operandsAndOperators: Array<number | string> = inputString.split(" ").map((token: string) => {
+        return isNaN(Number(token))
             ? token
             : Number(token);
-        console.log(parsedToken)
-        return parsedToken;
     });
-    console.log(operandsAndOperators)
 
-    const stack: number[] = [];
+    if (operandsAndOperators
+        .filter(x => typeof x === "number")
+        .length === operandsAndOperators
+        .filter(x => typeof x === "string")
+        .length + 1) {
 
-    operandsAndOperators.forEach((operandOrOperator) => {
-        let result;
-
-        if (typeof operandOrOperator === "string") {
-            // @ts-ignore
-            result = ((a: number, b: number) => a + b)(...stack.splice(-2));
-        } else result = operandOrOperator;
-        stack.push(result);
-    });
```

Comment:
First of all, you used @ ts-ignore because the compiler reported the error: "TS2556: A spread argument must either have a tuple type or be passed to a rest parameter."
It means that the compiler does not know if the received data matches the requirements of the function (for example, if there are enough arguments to be returned).
The easiest way is to make sure he gets what he needs. As long as you are sure it is.

Use type assertion if you are sure args will always be 2 elements tuple:
"result = ((a: number, b: number) => a! + b!) (... stack.splice (-2) as [number, number]);" 

Second thing, you should check which operator you came across. In your version it only checks if it's an operator (if it's a string), and you sum operands each time. You need to validate what operator it is and take appropriate action.